backend/public-graph/graph/resolver.go

```diff
@@ -2200,12 +2195,7 @@ func (r *Resolver) ProcessBackendPayloadImpl(ctx context.Context, sessionSecureI
 
 		var mappedStackTrace *string
 		var structuredStackTrace []*privateModel.ErrorTrace
-
-		if settings.EnableEnhancedErrors {
-			mappedStackTrace, structuredStackTrace, err = r.Store.EnhancedStackTrace(ctx, v.StackTrace, workspace, &project, errorToInsert)
-		} else {
-			structuredStackTrace, err = r.Store.StructuredStackTrace(ctx, v.StackTrace)
-		}
+		mappedStackTrace, structuredStackTrace, err = r.Store.EnhancedStackTrace(ctx, v.StackTrace, workspace, &project, errorToInsert)
```

Comment:
could we avoid a noisy error log in case the customer does not have github integrated / a service name defined?